Makefile

```diff
@@ -165,15 +167,21 @@ protokube-build-in-docker: protokube-builder-image
 	docker run -it -e VERSION=${VERSION} -v `pwd`:/src protokube-builder /onbuild.sh
 
 protokube-image: protokube-build-in-docker
-	docker build -t ${DOCKER_REGISTRY}/protokube:${PROTOKUBE_TAG} -f images/protokube/Dockerfile .
+	docker build -t protokube:${VERSION} -f images/protokube/Dockerfile .
+
+protokube-export: protokube-image
+	mkdir -p .build/dist/images
+	docker save protokube:${VERSION} | gzip -c  > .build/dist/images/protokube.tar.gz
+	(sha1sum .build/dist/images/protokube.tar.gz | cut -d' ' -f1) > .build/dist/images/protokube.tar.gz.sha1
 
 protokube-push: protokube-image
-	docker push ${DOCKER_REGISTRY}/protokube:${PROTOKUBE_TAG}
+	docker tag protokube:${VERSION} ${DOCKER_REGISTRY}/protokube:${VERSION}
+	docker push ${DOCKER_REGISTRY}/protokube:${VERSION}
```

Comment:
So I am confused.  We are pushing to which docker registry with the release?  Also we need to figure out ECR as well, probably where we want to host this from.